src/Symfony/Component/Form/FormInterface.php

```diff
@@ -45,7 +47,7 @@ public function getParent();
      * @param string|null              $type    The child's type, if a name was passed
      * @param array                    $options The child's options, if a name was passed
      *
-     * @return self
+     * @return $this
```

Comment:
```suggestion
     * @return static
```

Reply:
Your proposition is not consistent with what we do in core, the policy is to use `@return $this` for fluent interface without strict return type hint.

Reply:
Ah yes, true for `getParent()` thanks!

Reply:
it should even stay `self`. There is no guarantee that it will be the same implementation of the interface (the parent of a Button will never be a button). So that change must be reverted entirely.

Reply:
Good catch, reverted.